misc/services/mobile-webui/mobile-webui-frontend/src/apps/huManager/containers/HUBulkActionsScreen.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+
+import { trl } from '../../../utils/translations';
+import { getHandlingUnitInfoFromGlobalState } from '../reducers';
+import { pushHeaderEntry } from '../../../actions/HeaderActions';
+import ButtonWithIndicator from '../../../components/buttons/ButtonWithIndicator';
+import BarcodeScannerComponent from '../../../components/BarcodeScannerComponent';
+import { toQRCodeDisplayable } from '../../../utils/qrCode/hu';
+import * as api from '../api';
+import { toastError, toastNotification } from '../../../utils/toast';
+
+const HUBulkActionsScreen = () => {
+  const dispatch = useDispatch();
+  const history = useHistory();
```

Comment:
pls adapt it to latest new_dawn_uat changes, i.e.
useScreenDefinition(screenId:'HUBulkActionsScreen'.....)